goagen/gen_client/cli_generator.go

```diff
@@ -693,4 +732,99 @@ func hasFlag(name string) bool {
 	}
 	return false
 }
-`
+
+func jsonVal(val string) *interface{} {
+	var t interface{}
+	err := json.Unmarshal([]byte(val), t)
+	if err != nil {
+		panic(err)
```

Comment:
we shouldn't panic if these fail. It means a bad input from the user not a bug.
Instead these should return an error that bubbles all the way up (understood this is going to trigger more changes).